src/apps/Console/handlers/act.js

```diff
@@ -2,9 +2,17 @@ import { parseMethodCall } from '../console-utils'
 import { encodeFunctionCallFromSignature } from '../web3-encoding-utils'
 
 export default async function actHandler(params, { wrapper }) {
-  const [selectedAgentInstance, targetAddress, methodWithArgs] = params
+  const [
+    selectedAgentInstance,
+    targetAddress,
+    methodWithArgs,
+    ethAmount = 0,
+  ] = params
   const [methodName, methodParams, methodArgs] = parseMethodCall(methodWithArgs)
-  const methodSignature = `${methodName}(${methodParams.join(',')})`
+
+  const methodSignature = methodParams
+    ? `${methodName}(${methodParams.join(',')})`
+    : `${methodName}()`
```

Comment:
I don’t think `methodParams` can be falsy? https://github.com/aragon/aragon/blob/0c0294252612f47503d99a42c7b5d9d4d532c635/src/apps/Console/console-utils.js#L80-L82

Reply:
Actually, it can be: https://github.com/aragon/aragon/blob/0c0294252612f47503d99a42c7b5d9d4d532c635/src/apps/Console/console-utils.js#L43-L53

Reply:
Oooh you’re right!